File: backend/src/main/java/org/jdk/project/dto/config/AudioDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class AudioDto {
  private Long id;
  private String audioKey;
  private String audioType;
  private String languageType;
  private String textInfo;
  private String audioValue;
  private String roomCode;
  private String deptName;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/AudioDto.kt
package org.jdk.project.dto.config

data class AudioDto(
    var id: Long? = null,
    var audioKey: String? = null,
    var audioType: String? = null,
    var languageType: String? = null,
    var textInfo: String? = null,
    var audioValue: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var remark: String? = null,
)
